webapp/components/create_comment.jsx

```diff
@@ -68,30 +69,34 @@ class CreateComment extends React.Component {
         this.hidePostDeletedModal = this.hidePostDeletedModal.bind(this);
 
         PostStore.clearCommentDraftUploads();
+        MessageHistoryStore.resetAllHistoryIndex();
```

Comment:
You probably want to change this to just reset the history for the RHS since it'll reset the post history index for the centre channel. I doubt anyone will notice, but it is technically incorrect to reset that here.